Pomodoro.DataAccess/Migrations/20230205105345_InitIdentity.cs

```diff
@@ -0,0 +1,305 @@
+﻿using System;
```

Comment:
If we don't have functionality to apply migration at startup(only for Dev configuration) - it would be nice to add. I encounter it all the time on the projects :)